how-to-guides/celestia-app-security.md

```
@@ -0,0 +1,282 @@
+    server all drop
+```
+
+::: tip
+
+Currently bind to eth0, make sure you bind it to the correct interface or bind all
+
+```interface any world```
+
+:::
+
+#### Key Configuration Details
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)